witness/src/Dscp/Witness/CLI.hs

```diff
@@ -70,6 +73,14 @@ netServParamsParser :: Parser NetServParams
 netServParamsParser =
     NetServParams <$> peersParser <*> ourZTNodeIdParser <*> optional internalZTNodeIdParser
 
+----------------------------------------------------------------------------
+-- Metrics server
+----------------------------------------------------------------------------
+
+metricsServerParser :: Parser (Maybe Endpoint)
```

Comment:
If you define `metricsServerParser` without `optional`, it'll be more flexible to use `metricsServerParser` from the outside -- you can always add `optional` in the direct place of usage.